adhocracy4/comments_async/api.py

```diff
@@ -111,3 +111,11 @@ def destroy(self, request, *args, **kwargs):
         comment.save()
         serializer = self.get_serializer(comment)
         return Response(serializer.data)
+
+    def block(self, request, *args, **kwargs):
+        comment = self.get_object()
+        if self.request.user == comment.creator:
+            comment.is_blocked = True
+        comment.save()
+        serializer = self.get_serializer(comment)
+        return Response(serializer.data)
```

Comment:
What is this doing?

Reply:
I added this because I couldn't see is_blocked (out of desperation), but didn't really think it was right. It's gone :)